src/components/CodeExample.js

```diff
@@ -111,25 +110,32 @@ const CodeExampleEl = styled.div`
   }
 `;
 
-const CopyIconWrapper = styled.div`
-  display: block;
-  position: relative;
+const CopyWrapperEl = styled.div`
   cursor: pointer;
+  padding: 0.75rem 1rem;
```

Comment:
I thought the click-to-copy target was a little small, so I moved the padding off `TitleEl` and onto the copy button. Had to shuffle some other margins/padding to make it work